src/winetricks

```diff
@@ -262,6 +262,13 @@ w_warn()
     unset _W_timeout
 }
 
+# Output debug message to stdin if debug variable is non-zero
+w_debug()
+{
+    # shellcheck disable=SC2154
+    [ -n "$debug" ] && printf 'DEBUG: %s\n' "$1"
```

Comment:
A) debug is wayyy too generic of a name.
B) Convention (https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L135) would be `W_DEBUG`

Reply:
Also, please assign $1 to a variable (msg or similar)

Reply:
> Also, please assign $1 to a variable (msg or similar)

Resolved in: https://github.com/Winetricks/winetricks/commit/95c76986c10da49fcdf9f0f9e9c576e500338787

Reply:
> A) debug is wayyy too generic of a name.

Is that a bad thing?

Reply:
> B) Convention (https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L135) would be W_DEBUG

Resolved in https://github.com/Winetricks/winetricks/pull/1339/commits/8d806013f8823469affb69afb807d18090e9ca95